client/my-sites/site-settings/settings-jetpack/main.jsx

```diff
@@ -1,23 +1,27 @@
 import { localize } from 'i18n-calypso';
 import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
+import { connect, useSelector } from 'react-redux';
+import AdvancedCredentials from 'calypso/components/advanced-credentials';
 import DocumentHead from 'calypso/components/data/document-head';
 import QueryRewindState from 'calypso/components/data/query-rewind-state';
 import QuerySitePurchases from 'calypso/components/data/query-site-purchases';
 import EmptyContent from 'calypso/components/empty-content';
 import FormattedHeader from 'calypso/components/formatted-header';
 import Main from 'calypso/components/main';
 import SidebarNavigation from 'calypso/my-sites/sidebar-navigation';
-import JetpackCredentials from 'calypso/my-sites/site-settings/jetpack-credentials';
+//import JetpackCredentials from 'calypso/my-sites/site-settings/jetpack-credentials';
 import JetpackDevModeNotice from 'calypso/my-sites/site-settings/jetpack-dev-mode-notice';
 import SiteSettingsNavigation from 'calypso/my-sites/site-settings/navigation';
 import { siteHasScanProductPurchase } from 'calypso/state/purchases/selectors';
+import getCurrentQueryArguments from 'calypso/state/selectors/get-current-query-arguments';
 import isRewindActive from 'calypso/state/selectors/is-rewind-active';
 import isSiteFailedMigrationSource from 'calypso/state/selectors/is-site-failed-migration-source';
 import { isJetpackSite } from 'calypso/state/sites/selectors';
 import { getSelectedSite, getSelectedSiteId } from 'calypso/state/ui/selectors';
 
 const SiteSettingsJetpack = ( { site, siteId, siteIsJetpack, showCredentials, translate } ) => {
+	const { host, action } = useSelector( getCurrentQueryArguments );
```

Comment:
Let's not mix `connect` and `useSelector` in one component. Either add a new selector call to the `connect` map function, or convert the entire component to hooks.